### DISCUSSION TITLE
Why was the helm chart removed from the docs?

### DISCUSSION THREAD

**Discussion by Marfien** [0xERR0R/blocky#1636] [Q&A] [answered]
I'd like to install blocky in some combination with coredns on my homelab cluster. I stumbled on the docs from v0.23 where there is a link to helm chart maintained by truecharts (?). Is there a specific reason, why it is not mentioned anymore in the docs? Is smth. wrong with it?

**Comment by ThinkChaos** [comment, marked as answer]
See the discussion in #847.  
TL;DR is we had bad feedback about it so it's only mentioned for TrueNAS now.
